unit_tests/README.md

```diff
@@ -0,0 +1,63 @@
+# DTS scripts unit tests
```

Comment:
I love the idea of more testing like this in general.
Have you looked into some framework if they are applicable here, such as: https://github.com/bats-core/bats-core

Reply:
Not yet.

Reply:
https://github.com/Dasharo/dasharo-issues/issues/719

Reply:
@artur-rs cc